### ISSUE TITLE
OrthoSlicer3D interactions broken on mac

### ISSUE THREAD

**Issue by agramfort** [nipy/nibabel#433]
when playing with OrthoSlicer3D I observe that the interactions (clicking to change slices) does not work with osx backend (no reactions to click) and half broken when using qt backend (only 1 of the three slices responds to clicks)

can someone reproduce?

I am testing using matplotlib 1.4.3 and python 2.7

**Comment by matthew-brett** [comment]
Yup, I get the same effect on OSX backend, in matplotlib 1.4.3 and 1.5.1.  I'm believe this is due to the fact that the OSX backend doesn't implement the relevant animation calls - see : https://github.com/matplotlib/matplotlib/issues/531

There's a new PR for matplotlib that just addressed this : https://github.com/matplotlib/matplotlib/pull/6178

**Comment by matthew-brett** [comment]
Yes, I get the same behavior for pyqt4 on homebrew at least.   All works well for me with the TkAgg backend.  @Eric89GXL - what backend were you testing on?  Do you get the same problems as we do?

**Comment by Eric89GXL** [comment]
Linux, qt backend. Oh OSX...

**Comment by matthew-brett** [comment]
More investigation : only one window updating for qt4, matplotlib 1.4.3 or 1.5.1, Python 2.7.   Seems to work correctly for qt5, matplotlib 1.4.3 or 1.5.1, Python 3.5.

**Comment by Eric89GXL** [comment]
@matthew-brett last time I checked it worked fine over here.

**Comment by agramfort** [comment]
I also see only one window updating for qt4

**Comment by matthew-brett** [comment]
Alex - what do you get with qt4 if you use IPython and start with `%matplotib qt` ?

https://github.com/matplotlib/matplotlib/issues/5517

**Comment by agramfort** [comment]
I see only one image reacting

**Comment by effigies** [comment]
Hi all, is this still a problem for anybody? Using the latest OSX, I was able to run `img.orthoview()` and move around quite easily.
